src/tests/ftest/pool/management_race.yaml

```diff
@@ -1,3 +1,8 @@
+"""
```

Comment:
Do we need copyright for yaml? I don't see it in any other yaml file.

Reply:
Yes. When I commit this PR, it start showing error about the copyright on the yaml file.

Reply:
done.